web/pgadmin/misc/cloud/biganimal/__init__.py

```diff
@@ -261,11 +263,11 @@ def _check_admin_permission(self):
                 return True
         return False
 
-    def get_regions(self):
+    def get_regions(self, provider_id):
         """Get regions"""
-        _url = "{0}/{1}".format(
+        _url = '{0}/cloud-providers/{1}/regions'.format(
```

Comment:
Fix Sonarlint issue - Change this code to not construct the URL from user-controlled data.